.github/workflows/audit-fix.yml

```
@@ -0,0 +1,26 @@
+# This workflow will automatically run `npm audit fix` every Thursday morning and open a PR if there are changes
+name: Audit fix
+on:
+  schedule:
+    - cron: '0 6 * * 4'
+jobs:
+  audit_fix:
+    runs-on: ubuntu-22.04
+    name: Run npm audit fix
+    steps:
+      - name: Checkout latest
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Run npm audit fix
+        run: npm audit fix
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          base: main
+          title: npm audit fix
```

```suggestion
          title: chore: npm audit fix
```
Or some other conventional commit prefix